libs/common/src/types/api-error-response-body.ts

```diff
@@ -0,0 +1,7 @@
+import { ApiErrorCode } from './api-error-code';
+
+export type ApiErrorResponseBody = {
```

Comment:
do we want to commonise this or not part of scope?

Reply:
I think this is as commonised as it can realistically get for now. Commonising further would involve looking at every controller which I think it a bit excessive for this PR, but it's definitely something we may want to try and encourage for new controllers we add going forward